.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+on:
+  create:
+    tags:
+       - 'v*.*'
+    
+jobs:
+  docker_build_push:
+    env:
+      GIT_REF: ${{ github.ref }}
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Scala
+      uses: actions/setup-java@v2
+      with:
+        distribution: adopt
+        java-version: 11
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Show output of sed to build.sbt command
+      run: |
+        VERS_TAG=$(echo $GIT_REF | sed 's/refs\/tags\/v//g')
+        VERS_TAG="\"$VERS_TAG"\"
+        sed "s/version := *.*/version := $VERS_TAG/" build.sbt
+    - name: Write Tag to Scala Build Version
+      run: |
+        VERS_TAG=$(echo $GIT_REF | sed 's/refs\/tags\/v//g')
+        VERS_TAG="\"$VERS_TAG"\"
+        sed -i "s/version := *.*/version := $VERS_TAG/" build.sbt
+    - name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+         username: ${{ secrets.DOCKERHUB_USERNAME }}
+         password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build
+      run: sbt docker:publishLocal
+    - name: Push
+      run: |
+        vers_num="$(grep 'version :=' build.sbt | head -n1 | sed 's/.*"\(.*\)".*/\1/')"
+        local_param="csvw-check:$vers_num"
+        push_param="gsscogs/csvw-check:v$vers_num"
+        docker tag $local_param $push_param
```

Comment:
It'd be nice if we could tag the latest build with `latest`. Could you make it push this as the `latest` to docker hub so long as the git tag doesn't contain `RC`?

Reply:
Without RC: https://github.com/GSS-Cogs/csvw-check/runs/4432000313?check_suite_focus=true

- Works as expected.

With RC: https://github.com/GSS-Cogs/csvw-check/runs/4432117266?check_suite_focus=true

- Works as expected.

Verifiable here: https://hub.docker.com/r/gsscogs/csvw-check/tags